python/mxnet/numpy/linalg.py

```diff
@@ -232,7 +232,7 @@ def svd(a):
     return _mx_nd_np.linalg.svd(a)
 
 
-def cholesky(a):
+def cholesky(a, lower=True):
```

Comment:
Seems official numpy does not support lower?
https://docs.scipy.org/doc/numpy/reference/generated/numpy.linalg.cholesky.html

Reply:
Official numpy not support 'lower'. At the backend, the implementation of 'cholesky' is to call 'potrf' in the file la_op-inl.h, and it needs a parameter 'lower'. I will use this parameter directly in the return statement, just like '_api_internal.cholesky(a, True)' and interface will be the same as official numpy.